test/dialect/mssql/test_reflection.py

```diff
@@ -241,6 +242,63 @@ def test_cross_schema_fk_pk_name_overlaps(self):
             ],
         )
 
+    @testing.provide_metadata
+    def test_pk_fk_column_order(self):
```

Comment:
As I mentioned in https://github.com/sqlalchemy/sqlalchemy/issues/5661#issuecomment-713138572 I think this could go in `lib\sqlalchemy\testing\suite\test_reflection.py`

Reply:
Thanks for the suggestion, I will get his moved over to the general reflection tests.

Reply:
Thanks for the suggestion, I will get this moved over to the general reflection tests.

Reply:
I have moved the test to the suggested file. Also, when running the tests locally I found that SQLite did not respect the primary key column order either. I have added an additional change for SQLite to sort the columns by the primary key order.